fe2-android/app/src/main/java/com/github/dedis/student20_pop/detail/fragments/LaoDetailFragment.java

```diff
@@ -161,27 +152,16 @@ private void setupEventListAdapter() {
     expandableListView.expandGroup(1);
   }
 
-  private void setupSwipeRefresh() {
-    //    mLaoDetailFragBinding.swipeRefresh.setOnRefreshListener(
-    //        () -> {
-    //          mWitnessListViewAdapter.notifyDataSetChanged();
-    //          mEventListViewEventAdapter.notifyDataSetChanged();
-    //          if (getFragmentManager() != null) {
-    //            getFragmentManager().beginTransaction().detach(this).attach(this).commit();
-    //          }
-    //          mLaoDetailFragBinding.swipeRefresh.setRefreshing(false);
-    //        });
-  }
-
   private void showHideProperties(Boolean show) {
-    mLaoDetailFragBinding.propertiesLinearLayout.setVisibility(show ? View.VISIBLE : View.GONE);
+    mLaoDetailFragBinding.propertiesLinearLayout.setVisibility(Boolean.TRUE.equals(show) ? View.VISIBLE : View.GONE);
   }
 
   private void editProperties(Boolean edit) {
-    mLaoDetailFragBinding.editPropertiesLinearLayout.setVisibility(edit ? View.VISIBLE : View.GONE);
+    mLaoDetailFragBinding.editPropertiesLinearLayout.setVisibility(Boolean.TRUE.equals(edit) ? View.VISIBLE : View.GONE);
```

Comment:
What is the purpose of this construct Boolean.TRUE.equals ?

Reply:
A `Boolean` may be `null` so `Boolean.TRUE.equals()` returns false in case `edit` is `null`.

Reply:
Thank you